datalad/metadata/extractors/datalad_core.py

```diff
@@ -41,6 +42,28 @@ def _get_dataset_metadata(self):
             obj['@context'] = obj['definition']
             del obj['definition']
         obj['@id'] = self.ds.id
+        subdsinfo = [{
+            # this version would change anytime we aggregate metadata, let's not
+            # do this for now
+            #'version': sds['revision'],
+            'type': sds['type'],
+            'name': sds['gitmodule_name'],
```

Comment:
would this be the place to eventually be able to ingest some "relationship" between datasets as [FAIR I3](https://www.go-fair.org/fair-principles/i3-metadata-include-qualified-references-metadata/) -- created a dedicated #2290

Reply:
I think that IS the information we have on the downward dataset relationships. Anything in adding would require the presence of a subdataset, which we can neither require nor guarantee.